### PR TITLE
New package: dumpasn1

### PR DESCRIPTION
**Pull request by felix**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (amd64-glibc)

### CODE REVIEW

File: srcpkgs/dumpasn1/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+short_desc="ASN.1 data display"
+maintainer="Felix Hanley <felix@userspace.com.au>"
+license=BSD
+homepage="https://www.cs.auckland.ac.nz/~pgut001/"
+distfiles="https://www.cs.auckland.ac.nz/~pgut001/dumpasn1.c"
+checksum=8ce8fdbf2e9b11d410b0ab4e44a6b3f89c27080113f051ec1054d230e050a0b8
+
+do_fetch() {
+	mkdir -p ${wrksrc}
+	$fetch_cmd "${homepage}/dumpasn1.c"
```
Comment (classabbyamp) [inline srcpkgs/dumpasn1/template:15]: you can use `skip_extraction` instead of this

**Review by classabbyamp — CHANGES_REQUESTED** [review]
> Software need to be used in version announced by authors as ready to use by the general public - usually called releases. Betas, arbitrary VCS revisions, templates using tip of development branch taken at build time and releases created by the package maintainer won't be accepted.

**Comment by ahesford** [comment]
Given the comments at the start of the downloaded source file, the quoted passage about stylistic requirements for acceptance as a Void package, and the fact that the program author doesn't even make an attempt to package this for distribution, I don't believe this is suitable for Void.

**Comment by felix** [comment]
It is is [reasonably common](https://pkgs.org/search/?q=dumpasn1) but Void missing it is fine I suppose.

**Pull request closed without merging**